projects/cashmere/src/lib/typeahead/typeahead-item/typeahead-item.component.ts

```diff
@@ -1,21 +1,34 @@
-import {Component, EventEmitter, Input, Output} from '@angular/core';
+import {Component, EventEmitter, Input, Output, Host} from '@angular/core';
+import {TypeaheadComponent} from '../typeahead.component';
```

Comment:
Can this circular dependency now be removed?

Reply:
Change complete!